docs/src/modules/components/Notifications.js

```diff
@@ -96,7 +97,7 @@ class Notifications extends React.Component {
         }
         action={
           <Button size="small" color="secondary" onClick={this.handleClose}>
-            Close
+            {t('close')}
```

Comment:
Should this perhaps be `closeButton` to give translators more context, and to help distinguish the meaning from "close" as in "nearby"?

```suggestion
            {t('closeButton')}
```